xml/art_sles_rpi_quick.xml

```diff
@@ -13,7 +13,7 @@
 <article role="sbp" xmlns="http://docbook.org/ns/docbook"
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
-  xml:id="art-rpiquick" xml:lang="en">
+  xml:id="art-raspi" xml:lang="en">
```

Comment:
The late "Raspbian OS" (now Raspberry Pi OS) had an additional `b` between p and i.
I think the "was there a b in SUSE's particular abbreviation or not?" issue could nicely be sidestepped by using the full product name:
```suggestion
  xml:id="art-raspberry-pi" xml:lang="en">
```